proto/cosmos/gov/v1/genesis.proto

```diff
@@ -17,10 +17,15 @@ message GenesisState {
   repeated Vote votes = 3;
   // proposals defines all the proposals present at genesis.
   repeated Proposal proposals = 4;
-  // params defines all the paramaters of related to deposit.
+  // Deprecated: Prefer to use `params` instead.
+  // deposit_params defines all the paramaters of related to deposit.
   DepositParams deposit_params = 5;
```

Comment:
```suggestion
  DepositParams deposit_params = 5 [deprecated=true];
```

Reply:
This will just add a comment to the generated code like `Deprecated:...`, already added the description about deprecation to the field in proto.

Reply:
https://developers.google.com/protocol-buffers/docs/proto3

its the standard way, you can find it in the above link if you search for deprecated. It also recommends changing it to reserved so that users don't use it.

Reply:
Yes for adding the `deprecated` option, but let's not change it to `reserved`. We once did that and it broke some JS clients and then reverted.